THIRD-PARTY-NOTICES

```diff
@@ -147,6 +201,36 @@ SOFTWARE.
 
 -----------
 
+The following NPM package may be included in this product:
+
+ - safer-buffer@2.1.2
```

Comment:
where these deps just added from an npm install?

Reply:
hm weird, I did install jambo just as a dep change to test the action, then removed it and did `npm install` again. That's what came up. I just revert package-lock.json back to head of develop branch. seem ok now